frontend/mulighetsrommet-veileder-flate/src/views/tiltaksgjennomforing-detaljer/ViewTiltakstypeDetaljer.tsx

```diff
@@ -19,9 +19,19 @@ import { useNavigerTilDialogen } from '../../hooks/useNavigerTilDialogen';
 import TiltaksgjennomforingsHeader from '../../layouts/TiltaksgjennomforingsHeader';
 import { capitalize, formaterDato } from '../../utils/Utils';
 import styles from './ViewTiltaksgjennomforingDetaljer.module.scss';
+import { environments } from '../../env';
 
 const whiteListOpprettAvtaleKnapp: Tiltakstyper[] = ['Midlertidig lønnstilskudd'];
 
+function lenkeTilOpprettAvtaleForEnv(): string {
+  const env: environments = import.meta.env.VITE_ENVIRONMENT;
+  const baseUrl =
+    env === 'production'
+      ? 'https://tiltaksgjennomforing.intern.nav.no/'
+      : 'https://tiltaksgjennomforing.dev.intern.nav.no/';
+  return baseUrl + 'tiltaksgjennomforing/opprett-avtale';
```

Comment:
Megapirk, men nevner det hvis du ikke er klar over det
Du kan bruke string interpolation med backticks slik 
```js
`${baseUrl}tilbakesgjennomforing/opprett-avtale\`
```

Så slipper du pluss-tegnet for å konkatinere strengene